### PR TITLE
Enable JRuby testing for Active Job

### PR DESCRIPTION
**Pull request by maclover7**
### Summary

All tests pass for me locally.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -69,6 +69,11 @@ matrix:
       env:
         - "JRUBY_OPTS='--dev -J-Xmx1024M'"
         - "GEM='am'"
+    - rvm: jruby-9.1.5.0
+      jdk: oraclejdk8
+      env:
+        - "JRUBY_OPTS='--dev -J-Xmx1024M'"
```
Comment (maclover7) [inline .travis.yml]: Per [your comment](https://github.com/rails/rails/pull/27074#discussion_r88349688) @matthewd, the only place we'd be able to "globally" specify this information is at the complete global level, so this env var would be set for all builds. If you're okay with that then I can make a patch to do that 😬

Comment (rafaelfranca) [inline .travis.yml]: Yeah, that is fine.

**Review by rafaelfranca — CHANGES_REQUESTED** [review]

**Comment by maclover7** [comment]
👍 updated @rafaelfranca

File: .travis.yml [hunk 2]
```diff
@@ -62,13 +64,15 @@ matrix:
...
         - "GEM='ap'"
     - rvm: jruby-9.1.5.0
       jdk: oraclejdk8
       env:
-        - "JRUBY_OPTS='--dev -J-Xmx1024M'"
         - "GEM='am'"
+    - rvm: jruby-9.1.5.0
+      jdk: oraclejdk8
+      env:
+        - "GEM='aj'"
```
Comment (matthewd) [inline .travis.yml]: :+1:

Let's lose the single quotes, and combine am and aj, as they are above.

**Comment by maclover7** [comment]
👍 updated @matthewd

**Pull request merged**